velodyne_laserscan/src/velodyne_laserscan.cpp

```diff
@@ -223,7 +223,13 @@ void VelodyneLaserScan::recvCallback(const sensor_msgs::msg::PointCloud2::Shared
         }
       else
         {
-          RCLCPP_WARN(get_logger(), "VelodyneLaserScan: PointCloud2 fields in unexpected order. Using slower generic method.");
+          // TODO(clalancette): This can be simplified when RCLCPP_WARN_ONCE is available
```

Comment:
Linked to https://github.com/ros2/rclcpp/issues/797

Reply:
Actually, never mind.  We have `RCLCPP_WARN_ONCE`, we just don't have the `THROTTLE` versions.  Since we only need `WARN_ONCE` here, we are good.